package.json

```diff
@@ -26,6 +26,8 @@
     "@nestjs/core": "^7.0.0",
     "@nestjs/mongoose": "^7.0.2",
     "@nestjs/platform-express": "^7.0.0",
+    "class-transformer": "^0.3.1",
```

Comment:
Are you using a class-transformer?

Reply:
Yes

Reply:
I couldn't see any references. Can you point me to it?

Reply:
Check if class-transformer is required, if not, please remove.